src/DgmOctree.cpp

```diff
@@ -3052,16 +3023,16 @@ unsigned DgmOctree::executeFunctionForAllCellsAtLevel(	unsigned char level,
 	{
 		if (maxThreadCount == 0)
 		{
-			// retrieve the maximum number of threads
+			//retrieve the maximum number of threads
 			maxThreadCount = QThread::idealThreadCount();
 		}
-		if (maxThreadCount == 1)
-		{
-			// if only one thread should/cloud be used, the direct approach is more efficient
-			multiThread = false;
-		}
+		//if (maxThreadCount == 1)
```

Comment:
why removing this?

Reply:
I just merge these commit, CC point to this branch https://github.com/CloudCompare/CCCoreLib/tree/52667084b18f872d737f97d677f304f962e76f53

Reply:
Wow, I'm lost 😅 

There have been way too many back and forth...

Reply:
So I guess we should merge it as is 🤔 . Let's trust me-from-the-past 😅